server/tserver/src/main/java/org/apache/accumulo/tserver/ThriftClientHandler.java

```diff
@@ -1402,10 +1402,8 @@ public void loadTablet(TInfo tinfo, TCredentials credentials, String lock,
       synchronized (server.openingTablets) {
         synchronized (server.onlineTablets) {
 
-          // checking if this exact tablet is in any of the sets
-          // below is not a strong enough check
-          // when splits and fix splits occurring
-
+          // With splits occurring, checking if the current tablet
+          // is in any of the sets below may not be a strong enough check
```

Comment:
This is mostly just a reorder of works of the original comment. I think it would help to mention what the checks are doing, i.e. checking if the current tablet overlaps any other in the set.